### PR TITLE
minor readme updates

### PR DESCRIPTION
**Pull request by jakemulf**

### CODE REVIEW

**Comment by jakemulf** [comment]
I'm not sure what to put for some examples in here. I did find https://github.com/eugenp/tutorials/blob/master/libraries-data-2/src/main/java/com/baeldung/suanshu/SuanShuMath.java, is that applicable? If not can someone (@devinrsmith?) provide some examples?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,150 @@
...
+This extension fixes testcases and serialized the library.
+
+## Installation
+
+The Maven repository for this project can be found at https://mvnrepository.com/artifact/io.deephaven/SuanShu
+
+### pom.xml
+
+```
+<!-- https://mvnrepository.com/artifact/io.deephaven/SuanShu -->
```
Comment (devinrsmith) [inline README.md:12]: No need for comment IMO.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,150 @@
...
+    <groupId>io.deephaven</groupId>
+    <artifactId>SuanShu</artifactId>
+    <version>0.1.0</version>
+</dependency>
+```
+
+### Gradle
+
+```
+// https://mvnrepository.com/artifact/io.deephaven/SuanShu
```
Comment (devinrsmith) [inline README.md:23]: No need for comment IMO.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,150 @@
...
+implementation group: 'io.deephaven', name: 'SuanShu', version: '0.1.0'
```
Comment (devinrsmith) [inline README.md:24]: We should prefer the shorter form: `implementation 'io.deephaven:SuanShu:0.1.0'`

**Review by devinrsmith — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,2 +1,150 @@
 # SuanShu
-This extension fixes testcases and serialized the library.
\ No newline at end of file
+
+This extension fixes testcases and serialized the library.
+
+## Installation
+
+The Maven repository for this project can be found at https://mvnrepository.com/artifact/io.deephaven/SuanShu
+
```
[the remark this answers is not in the document]
Comment (jakemulf) [reply README.md]: Updated to use search.maven

File: README.md (hunk 1 above)
Comment (jakemulf) [reply README.md]: Removed

**Review by devinrsmith — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,2 +1,148 @@
 # SuanShu
-This extension fixes testcases and serialized the library.
\ No newline at end of file
+
+This extension fixes testcases and serialized the library.
```
Comment (chipkent) [inline README.md:3]:
```suggestion
SuanShu is a Java math library for numerical analysis, statistics, root finding, linear algebra, optimization, curve fitting, interpolation, regression, differential equation solvers, and more.  

This repository is a fork of the most recent Open Source SuanShu release (2012-06-06).  It has enhancements that include fixes to test cases, serialization, etc.  If you are interested in the commercially licensed version, see [https://nm.dev/](https://nm.dev/).
```

**Review by chipkent — CHANGES_REQUESTED** [review]

**Pull request merged**